actionview/test/template/resolver_patterns_test.rb

```diff
@@ -19,7 +19,7 @@ def test_should_return_template_for_declared_path
     assert_equal 1, templates.size, "expected one template"
     assert_equal "Hello custom patterns!", templates.first.source
     assert_equal "custom_pattern/path",    templates.first.virtual_path
-    assert_equal :html,                    templates.first.format
+    assert_nil templates.first.format
```

Comment:
`nil` is actually wrong here, but not because of this change. Custom formats don't _really_ work. The file this finds is `test/fixtures/custom_pattern/html/path.erb` and `extract_handler_and_format_and_variant` only understands the default format, so we get `nil`.